sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -0,0 +1,236 @@
+          offering: { id: true },
+          studentAppeal: { id: true },
+          triggerType: true,
+          studentAssessmentStatus: true,
+        },
+        studentAssessments: {
+          studentAssessmentStatus: true,
+          offering: { id: true },
+          studentAppeal: { id: true },
+        },
+        isArchived: true,
+        applicationStatus: true,
+      },
+      relations: {
+        currentAssessment: { offering: true, studentAppeal: true },
+        studentAssessments: {
+          offering: true,
+          studentAppeal: true,
+        },
+      },
+      where: {
+        id: application.id,
+        studentAssessments: {
+          triggerType: AssessmentTriggerType.OriginalAssessment,
+        },
+      },
+    });
+    const [originalAssessment] = applicationFromDb.studentAssessments;
+    const manualAssessment = applicationFromDb.currentAssessment;
+    expect(manualAssessment.id).toBe(responseReassessmentId);
+    expect(manualAssessment.triggerType).toBe(
+      AssessmentTriggerType.ManualReassessment,
+    );
+    expect(manualAssessment.studentAssessmentStatus).toBe(
+      StudentAssessmentStatus.Submitted,
+    );
+    expect(manualAssessment.offering.id).toBe(originalAssessment.offering.id);
+    expect(manualAssessment.studentAppeal.id).toBe(
+      originalAssessment.studentAppeal.id,
+    );
+
+    const student = await db.student.findOne({
+      select: { notes: true },
+      relations: { notes: true },
+      where: { id: application.student.id },
+    });
+    expect(student.notes).toContainEqual(
+      expect.objectContaining({
+        id: expect.any(Number),
+        description: payload.note,
+        noteType: NoteType.Application,
+        updatedAt: expect.any(Date),
+        createdAt: expect.any(Date),
+      }),
+    );
+  });
+
+  it(`Should throw unprocessable entity when the original assessment has ${StudentAssessmentStatus.Completed} status.`, async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource);
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Cancelled;
+    application.studentAssessments = [application.currentAssessment];
+    await db.application.save(application);
+
+    const endpoint = `/aest/assessment/application/${application.id}/manual-reassessment`;
+    const payload = {
+      note: "Testing manual reassessment with a cancelled assessment.",
+    };
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+      .expect({
+        statusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+        message: `Application original assessment expected to be '${StudentAssessmentStatus.Completed}' to allow manual reassessment.`,
+        error: "Unprocessable Entity",
+      });
+  });
```

Comment:
As per the implementation from line 155-184, this code seems to be redundant unless I am missing something.